tsdb/engine/tsm1/engine.go

```diff
@@ -1443,7 +1444,18 @@ func (e *Engine) deleteSeriesRange(seriesKeys [][]byte, min, max int64, removeIn
 			}
 
 			// Remove the series from the index.
-			if err := e.index.UnassignShard(string(k), e.id, ts); err != nil {
+			// TODO(edd): if a DROP MEASUREMENT, or even a DROP SERIES had been
```

Comment:
This comment isn't correct.  You can't remove from series file/index until after we've verified that no concurrent writes were in place (e.g. no values added to cache while this delete is running).